packages/manager/apps/web/client/app/hosting/hosting.routes.js

```diff
@@ -12,6 +14,11 @@ export default /* @ngInject */ ($stateProvider) => {
       tab: null,
     },
     resolve: {
+      availableOptions: /* @ngInject */ (WucOrderCartService, serviceName) =>
+        WucOrderCartService.getProductServiceOptions(
+          'webHosting',
+          serviceName,
+        ).catch(() => []),
```

Comment:
you dont want to show error page on error?

Reply:
@radireddy In that case no, I don't think it is necessary to block the whole hosting dashboard if options cannot be fetched